desc/geometry/curve.py

```diff
@@ -72,6 +72,9 @@ def __init__(
 
         modes_R, modes_Z = np.asarray(modes_R), np.asarray(modes_Z)
 
+        assert R_n.size == modes_R.size, "R_n size and modes_R must be the same size!"
```

Comment:
I don't think we need exclamation points on all of these

Reply:
we should write all of our error messages with a really angry tone so the users feel stupid when they do something wrong